src/app/content/highlights/components/HighlightStyles.tsx

```diff
@@ -296,6 +296,9 @@ export const GeneralCenterText = styled(GeneralTextWrapper)`
   text-align: center;
 `;
 
+// tslint:disable-next-line:variable-name
+export const GeneralCenterTextWrapper = styled.span``;
```

Comment:
do you really need this styled component with no styles?

Reply:
I've used this because `htmlMessage` is accepting `ComponentType`, but I can also use `() => <span/>`